rules/privatization/tests/Rector/ClassMethod/MakeOnlyUsedByChildrenProtectedRector/Fixture/skip_used_by_external_in_closure.php.inc

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Rector\Privatization\Tests\Rector\ClassMethod\MakeOnlyUsedByChildrenProtectedRector\Fixture;
+
+class SkipUsedByExternalInClosure
+{
+    public function run()
+    {
+    }
+}
+
+function () {
+    $obj = new SkipUsedByExternalInClosure();
+    $obj->run();
+};
+
+?>
```

Comment:
There should be also use case with parent class method, that has a child, and the child method call is used.

```php
abstract class SomeParent
{
    public function run()
    {
    }
}

final class ChildClass extends SomeParent
{
}

final class SkipSomeUser
{
     public function useIt(ChildClass $childClass)
     {
         // the method is used, so it should be skipped
         $childClass->run();
     }
}
```

Reply:
implemented, added fixture for it.